crates/orchestrator/src/shared/constants.rs

```diff
@@ -0,0 +1,8 @@
+/// Prometheus exporter default port
```

Comment:
@l0r1s We should extract these to re-use in other crates. Maybe in support, wdyt?

Reply:
Yes sure